### PR TITLE
Clarify batch_norm documentation to highlight that the dimensions for…

### PR DESCRIPTION
**Pull request by Netzeband**
... normalization depend on the shape of the tensor.

### CODE REVIEW

**Comment by Netzeband** [comment]
I signed it!

**Comment by Netzeband** [comment]
Yes I used a different e-mail address for the commit. I cannot add this address to the google account where I signed the contract.

**Comment by drpngx** [comment]
Could you change the email address on the commit?

**Comment by Netzeband** [comment]
Ok I changed the e-mail address in my commit. However this is not a standard procedure with git and requires to rewrite all commits. As far as I can see, this pull-request just contains my change, so I think it should work now.

**Comment by drpngx** [comment]
OK, maybe let's wait until the next refresh.

**Review by vincentvanhoucke — APPROVED** [review]

**Comment by drpngx** [comment]
@Netzeband Still broken. I don'y anything pushed. You can amend the commit with `--author` and push again.

**Comment by Netzeband** [comment]
I think this will go into the wrong direction, I will close the request and setup a new one, where not the whole commit history have been rewritten (due to the e-mail address change).

**Comment by Netzeband** [comment]
I created a new request: https://github.com/tensorflow/tensorflow/pull/15728
